articles/active-directory/develop/howto-add-app-roles-in-azure-ad-apps.md

```diff
@@ -20,13 +20,15 @@ ms.custom: aaddev
 
 Role-based access control (RBAC) is a popular mechanism to enforce authorization in applications. When using RBAC, an administrator grants permissions to roles, and not to individual users or groups. The administrator can then assign roles to different users and groups to control who has access to what content and functionality.
 
-Using RBAC with Application Roles and Role Claims, developers can securely enforce authorization in their apps with less effort.
+Using RBAC with Application Roles and Role Claims, developers can securely integrate authorization into their apps with less effort.
 
-Another approach is to use Azure AD Groups and Group Claims as shown in the [active-directory-aspnetcore-webapp-openidconnect-v2](https://aka.ms/groupssample) code sample on GitHub. Azure AD Groups and Application Roles are not mutually exclusive; they can be used in tandem to provide even finer-grained access control.
+Another approach using RBAC is to use Azure AD Groups and Group Claims, as shown in the [active-directory-aspnetcore-webapp-openidconnect-v2](https://aka.ms/groupssample) code sample on GitHub. 
```

Comment:
Thanks for the edit, but please add this back ". Azure AD Groups and Application Roles are not mutually exclusive; they can be used in tandem to provide even finer-grained access control."